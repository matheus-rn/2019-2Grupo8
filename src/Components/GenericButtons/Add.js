import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import Fab from '@material-ui/core/Fab';
import AddIcon from '@material-ui/icons/Add';
import { withRouter } from 'react-router-dom'

import '../Feed/feed.css';

const styles = theme =>({
  
  fab: {
    top: 'auto',
    right: 20,
    bottom: 65,
    left: 'auto',
    position: 'fixed',
    margin: theme.spacing.unit,
  },
  
});

const ButtonSizes = ({ classes, history }) => {
  return (
      <Fab onClick={() => history.push('/RegisterMonitoring')} size="medium" style={{backgroundColor: '#1DA1F2',
          color: 'white'}} aria-label="Add" className={classes.fab}>
          
          <AddIcon />
      </Fab>
  );
}

ButtonSizes.propTypes = {
  classes: PropTypes.object.isRequired,
};
const materialUiStyles = withStyles(styles)(ButtonSizes)
export default withRouter(materialUiStyles)