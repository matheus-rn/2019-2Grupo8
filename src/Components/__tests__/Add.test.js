import React from 'react';
import { shallow } from 'enzyme';
import Add from '../GenericButtons/Add';
import toJson from 'enzyme-to-json';

describe('Testing Add component', () => {
    it('Test if Add renders correctly', () =>{
        const tree = shallow(<Add/>);
        expect(toJson(tree)).toMatchSnapshot();
    });
});