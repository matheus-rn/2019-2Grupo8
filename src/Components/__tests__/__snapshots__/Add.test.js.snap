// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Add component Test if Add renders correctly 1`] = `
<Route>
  <Component />
</Route>
`;
